account_cutoff_start_end_dates/README.rst

```diff
@@ -0,0 +1,124 @@
+=======================
+Account Cut-off Prepaid
+=======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--closing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-closing/tree/13.0/account_cutoff_prepaid
+    :alt: OCA/account-closing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-closing-13-0/account-closing-13-0-account_cutoff_prepaid
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/89/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows you to easily compute the prepaid revenue and prepaid expenses and to generate the related cutoff journal items. It uses the **Start Date** and **End Date** fields of journal entries (copied from the same fields of invoice lines).
+
+For
+example, if you have an insurance contrat for your company that run from April
+1st 2013 to March 31st 2014, you will enter these dates as start and end dates
+on the supplier invoice line. If your fiscal year ends on December 31st 2013,
+3 months of expenses are part of the 2014 fiscal year and should not be part of
+the 2013 fiscal year. So, thanks to this module, you will create a *Prepaid
+Expense* on December 31st 2013 and OpenERP will identify this expense with the
+3 months that are after the cut-off date and propose to generate the
+appropriate cut-off journal entry.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+On the form view of the company, in the *Configuration* tab, there is a *Cut-offs* section where you can configure:
+
+* the *Default Cut-off Journal*, which is the journal in which the cut-off journal items will be generated by default,
+* the *Default Account for Prepaid Revenue*,
+* the *Default Account for Prepaid Expense*.
```

Comment:
I can not find the Configuration tab definition with Cut-offs section in company view but just in **Invoicing > Configuration > Settings** as described in **account_cutoff_base** module

Reply:
Indeed. All configuration has been moved into Settings under the Accounting section within a Cut-offs sub section, which is a good thing. Unfortunately the documentation has not followed.